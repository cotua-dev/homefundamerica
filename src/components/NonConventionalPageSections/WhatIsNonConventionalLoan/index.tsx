import React, {
    FunctionComponent,
    ReactElement,
} from 'react';
import { whatIsNonConventionalLoanSection } from './WhatIsNonConventionalLoan.module.scss';
import * as sharedStyles from '../../sharedStyles/shared.module.scss';

export const WhatIsNonConventionalLoan: FunctionComponent = (): ReactElement => {
    return (
        <section id={whatIsNonConventionalLoanSection} className={sharedStyles.loanPositive}>
            <div className={sharedStyles.contentWrapper}>
                <h2>{`What is a Non-Conventional Loan?`}</h2>
                <p>
                    {`
                        A non-conventional loan is a mortgage that doesn't meet the
                        guidelines for a conventional loan set by Fannie Mae and
                        Freddie Mac. Often a loan is classified as non-conventional
                        because the loan amount exceeds the conventional limit, which
                        is $484,350 in most U.S counties. In addition to higher
                        loan amounts, non-conventional loans from Manhattan Banc can
                        offer expanded down-payment and credit qualification options.
                    `}
                </p>
            </div>
        </section>
    );
}
