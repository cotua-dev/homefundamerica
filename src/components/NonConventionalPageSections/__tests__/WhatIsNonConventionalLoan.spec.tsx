import React from 'react';
import {
    render,
    RenderResult,
} from '@testing-library/react';
import { WhatIsNonConventionalLoan } from '../WhatIsNonConventionalLoan';

describe('WhatIsNonConventionalLoan Component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<WhatIsNonConventionalLoan/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
