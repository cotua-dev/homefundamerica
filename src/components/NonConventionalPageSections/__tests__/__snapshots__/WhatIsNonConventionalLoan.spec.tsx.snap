// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatIsNonConventionalLoan Component renders 1`] = `
<section
  id="whatIsNonConventionalLoanSection"
>
  <div>
    <h2>
      What is a Non-Conventional Loan?
    </h2>
    <p>
      
                        A non-conventional loan is a mortgage that doesn't meet the
                        guidelines for a conventional loan set by Fannie Mae and
                        Freddie Mac. Often a loan is classified as non-conventional
                        because the loan amount exceeds the conventional limit, which
                        is $484,350 in most U.S counties. In addition to higher
                        loan amounts, non-conventional loans from Manhattan Banc can
                        offer expanded down-payment and credit qualification options.
                    
    </p>
  </div>
</section>
`;
